### DISCUSSION TITLE
bug: In older (pre-byzantium) ETH receipts `status` field is null

### DISCUSSION THREAD

**Discussion by LogvinovLeon** [wevm/viem#2038] [Question]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package Version

2.9.3

### Current Behavior

Currently the status type is 'success' | 'reverted'

### Expected Behavior

It should be `status?: 'success' | 'reverted'`

### Steps To Reproduce

Fetch this receipt: https://etherscan.io/tx/0x38f299591902bfada359527fa6b9b597a959c41c6f72a3b484807fbf52dc8abe

**Comment by jxom** [comment]
This is a breaking change, and I don't know whether it is worth it. Going to convert this to a discussion so it doesn't get lost.
